actions/slack-payload/src/libs/workflow-context/__tests__/main-context.test.ts

```diff
@@ -77,6 +78,38 @@ describe('getWorkflowContext method', () => {
       runId: 1,
       sha: 'sha',
       url: 'url',
+      author_email: mockedWorkflowInfo.author_email,
+    });
+  });
+
+  it('should fail', async () => {
+    jest.spyOn(fileClient, 'readFile').mockReturnValue('Bad JSON');
+    const payload = await getWorkflowContext(mockedToken);
+
+    expect(payload).toStrictEqual({
+      attachmentsIds: {
+        buildLogsArtifactId: 0,
+        workflowInfoArtifactId: 1,
+        testLogsArtifactId: 0,
+      },
+      checkSuiteId: 1,
+      conclusion: 'success',
+      environment: mockedWorkflowInfoOnFailure.environment,
+      jobsOutcome: {
+        failedJobSteps: [],
+        failureDeployCount: 0,
+        successDeployCount: 0,
+      },
+      name: 'name',
+      pullRequest: undefined,
+      repository: {
+        name: 'name',
+        url: 'url',
+      },
+      runId: 1,
+      sha: 'sha',
+      url: 'url',
+      author_email: mockedWorkflowInfoOnFailure.author_email,
     });
```

Comment:
Tests should not be more specific than they need to be

```suggestion
    expect(payload).toEqual(expect.objectContaining({
      environment: mockedWorkflowInfoOnFailure.environment,
      author_email: mockedWorkflowInfoOnFailure.author_email,
    }));
```